dao/src/main/java/org/thingsboard/server/dao/rule/RuleChainInfoDao.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright © 2016-2022 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.dao.rule;
+
+import org.thingsboard.server.common.data.page.PageData;
+import org.thingsboard.server.common.data.page.PageLink;
+import org.thingsboard.server.common.data.rule.RuleChainInfo;
+import org.thingsboard.server.common.data.rule.RuleChainType;
+
+import java.util.UUID;
+
+public interface RuleChainInfoDao {
+    /**
+     * Find rule chains using error statistics by tenantId, type and page link.
```

Comment:
please update java comment.
'Find rule chains using error statistics by tenantId, type and page link.' doesn't look as correct description.